about.html

```diff
@@ -7,57 +7,93 @@
   <title>Sprint Challenge - About</title>
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+  <link rel="stylesheet" href="css/about.css">
 
 </head>
 
 <body>
-    <div class="container about-page">
+    <header>
+        <span><img  class="aboutpic"src="https://github.com/LambdaSchool/web-Sprint-Challenge-User-Interface-and-Git/blob/master/img/lambda-black.png?raw=true"></span>
+        
+        
+            
+        <nav class="nav-text-1">
+                <a id="div1" href="index.html">Home</a>
+                <a id="div2" href="#div2section">About</a>
+                <a id="div3" href="#div3section">Products</a>
```

Comment:
For this you have the right idea if you had a single file application, this project has multiple files so the syntax to connect the href attribute to the about.html document will be -> ./ about.html  and home would be ./index.html.